dependencies.toml

```diff
@@ -69,21 +69,21 @@ joor = "0.9.14"
 json-unit = "2.38.0"
 jsoup = "1.16.1"
 junit4 = "4.13.2"
-junit5 = "5.9.3"
+junit5 = "5.10.0"
 junit-pioneer = "1.9.1"
 jwt = "4.4.0"
-kafka = "3.4.1"
-kotlin = "1.8.22"
-kotlin-coroutine = "1.7.1"
-ktlint-gradle-plugin = "11.4.0"
+kafka = "3.5.1"
+kotlin = "1.9.0"
+kotlin-coroutine = "1.7.3"
+ktlint-gradle-plugin = "11.5.0"
 logback12 = "1.2.11"
 logback14 = "1.4.7"
-micrometer = "1.11.1"
+micrometer = "1.11.2"
 micrometer13 = "1.3.20"
 mockito = "4.11.0"
 monix = "3.4.1"
 munit = "0.7.29"
-netty = "4.1.95.Final"
+netty = "4.1.96.Final"
```

Comment:
Would you mind resolving my TODO as well? 😉 https://github.com/line/armeria/blob/4edff55daaad24ba6f91e21bd2eabd0073a723ef/core/src/main/java/com/linecorp/armeria/client/AbstractDnsResolverBuilder.java#L530-L531

Reply:
Done! Thanks for reminding me 🙇

Reply:
Thanks!